salt/runners/net.py

```diff
@@ -826,6 +844,8 @@ def find(addr, best=True, display=_DEFAULT_DISPLAY):
     except IndexError:
         # no problem, let's keep searching
         pass
+    if isinstance(addr, (six.text_type, six.string_types)) and '::' in addr:
```

Comment:
We do have `salt.utils.network.is_ipv6`. Would it be better to use that instead?

Reply:
Yes, that's exactly what I needed. The irony is that I make some changes in that function relatively recent, but I forgot it exists. I don't know how you can keep track of everything @cachedout, while I already forgot what I forgot what I worked on just a couple of months ago.
I will change this, thanks for the hint!